fr.handbrake.ghb.Plugin.IntelMediaSDK.json

```diff
@@ -0,0 +1,111 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.IntelMediaSDK",
+    "branch": "1",
+    "runtime": "fr.handbrake.ghb",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//3.36",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "modules": [
+        {
+            "name": "intel-gmmlib",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/gmmlib/archive/intel-gmmlib-20.1.1.tar.gz",
+                    "sha256": "821755657cf51f59d8f3f443c99e3ec9f28d897ff65c219c6a119e4acb5a2ac7"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "build-options": {
+                "prefix" : "/app/extensions/IntelMediaSDK",
+                "make-args": [
+                    "VERBOSE=1"
+                ],
+                "make-install-args": [
+                    "VERBOSE=1"
+                ]
+            }
+        },
+        {
+            "name": "libva",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/libva/releases/download/2.7.1/libva-2.7.1.tar.bz2",
```

Comment:
It's in the runtime:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/19.08/elements/components/libva.bst

Reply:
Is it the exact same version?  Intel's drivers and mediasdk have had issues in the past running with untested versions of components.  I am packaging *exactly* the components specified in the Media SDK release https://github.com/Intel-Media-SDK/MediaSDK/releases/tag/intel-mediasdk-20.1.1

Reply:
It's at the bottom of that file:
```
sources:
- kind: git_tag
  url: github:intel/libva.git
  track: v2.5-branch
  ref: 2.5.0-0-ga7dc5b7f0501dcc5e88df1638b6f80d3b03f4c67
- kind: patch
  path: patches/libva/pkgconfig-driver-path.patch
```